cpp/models/abm/README.md

```diff
@@ -1,57 +1,147 @@
+
+### Migration Phase
+
+During the migration phase, each person may change their location. Migration follows complex [rules](../abm/migration_rules.cpp), considering the current location, time of day, and properties of the person (e.g. age). Some location changes are deterministic and regular (e.g. going to work), others are random (e.g. going to shopping or to a social event in the evening/on the weekend). When agents are infected, they are quarantined and cannot migrate. You can restrict some migration rules by allowing only a proportion of people to enter specific locations.
+
+Another way of migration we use in the simulation of Braunschweig (simulations/abm_braunschweig.cpp) is using trips. A trip consists of the ID of the person that performs this trip, a time point when this trip is performed and where the person is heading to. At the beginning of the simulation, a list with all trips is initialized and followed during the simulation. There can be different trips on the weekend than during the week, but other than that, the agents do the same trips every day. As before, agents that are in quarantine or in the hospital cannot migrate.
+
+## Get Started
+
+This section gives an introduction to how to use the ABM and set up your own simulation. For a quick overview, can find a full example in the [ABM minimal example](../../examples/abm_minimal.cpp) and a more detailed Doxygen documentation [here](https://scicompmod.github.io/memilio/documentation/index.html ). For a guide on installation and running the simulations and examples, see this [README](../../README.md).
+
+Every person in the ABM belongs to an AgeGroup, which we can define as the following. Note that every age group has to have values strictly smaller than num_age_groups.
+
+```cpp
+size_t num_age_groups         = 4;
+const auto age_group_0_to_4   = mio::AgeGroup(0);
+```
+
+The initial empty world is created with the number of age groups:
+
+```cpp
+auto world = mio::abm::World(num_age_groups);
+```
+
+We can set several general parameters, which you can find [here](../abm/parameters.h). Here is an example where we set the duration of the incubation period to 4 days:
+
+```cpp
+world.parameters.get<mio::abm::IncubationPeriod>() = 4.
+```
+
+To add a location to the world, we have to specify the kind of location.
+
+```cpp
+auto home = world.add_location(mio::abm::LocationType::Home);
+```
+
+People are added with an age. Then we have to assign them, so the world knows they can travel to this location.
+
+```cpp
+auto person = world.add_person(home, mio::AgeGroup(0));
+person.set_assigned_location(home);
+```
+
+For adding more people to the world, we create households. A Household holds a vector with HouseholdMembers, i.e. a vector with weighted age distribution from which the age of the Persons belonging to this Household can be calculated. A Household and the number of times it exists is gathered in a HouseholdGroup.
```

Comment:
```suggestion
For adding more people to the world, we create households. A Household holds a vector of HouseholdMembers, which in turn hold a weighted distribution, such that we can randomly draw the age of each Person belonging to the Household. To manage multiple Households of the same type, we can use a HouseholdGroup.
```

I tried to make this more descriptive of what each class does, instead of what it is in code. Please check the correctness of the suggestion, I only skimmed over the code.